Build/PackageFiles/Dotnet_precompiled/ItemTemplates-Isolated-NetFx_v4.x.nuspec

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.Azure.Functions.Worker.NetFx.ItemTemplates</id>
+    <version>1.0.$patchVersion$</version>
+    <authors>Microsoft</authors>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=2028464</licenseUrl>
+    <repository type="git" url="https://github.com/Azure/azure-functions-templates" />
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=848145</projectUrl>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Collection of templates for Azure Functions .NET Isolated using .NET Framework</description>
+    <language>en-US</language>
+    <packageTypes>
+      <packageType name="Template" />
+    </packageTypes>
+  </metadata>
+  <files>
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-CSharp-Isolated\QueueTriggerCSharp.cs" target="content\QueueTrigger-CSharp\QueueTriggerCSharp.cs" />
```

Comment:
Can likely follow the content format from this file. Easier to manage and review.
https://github.com/Azure/azure-functions-templates/blob/dev/Build/PackageFiles/Dotnet_precompiled/ItemTemplates-Isolated_v3.x.nuspec

Reply:
Shortened it to use the V3 content format, but would love a second pair of eyes to confirm that this new version still has all of the triggers present from https://github.com/Azure/azure-functions-templates/blob/dev/Build/PackageFiles/Dotnet_precompiled/ItemTemplates-Isolated_v4.x.nuspec